opm/parser/eclipse/Parser/MessageContainer.cpp

```diff
@@ -128,6 +128,15 @@ namespace Opm {
     }
 
 
+    void MessageContainer::appendMessages(const MessageContainer& msgContainer)
```

Comment:
I would prefer if the input argument was called `other` or something like that; because internally in this method we actually have *two* message containers in scope.